src/views/TechnicalAssistance/TaskList.test.tsx

```diff
@@ -172,10 +172,7 @@ describe('Trb Task List', () => {
 
     userEvent.click(confirm);
 
-    const alert = await screen.findByTestId('alert');
-    expect(alert).toHaveClass('usa-alert--success');
-
-    screen.getByText(
+    await screen.findByText(
```

Comment:
Not sure what this change is (or if it's related), but not enough to hold up a review!